src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -142,15 +142,15 @@ private void testInputValidation() {
         assertEquals(longEmail.substring(0, FieldValidator.EMAIL_MAX_LENGTH),
                      courseEditPage.fillInstructorEmail(longEmail));
         // Edit instructor
-        assertEquals(maxLengthEmail, courseEditPage.editInstructorEmail(maxLengthEmail));
+        assertEquals(maxLengthEmail, courseEditPage.editInstructorEmail(1, maxLengthEmail));
         assertEquals(longEmail.substring(0, FieldValidator.EMAIL_MAX_LENGTH),
-                     courseEditPage.editInstructorEmail(longEmail));
+                     courseEditPage.editInstructorEmail(1, longEmail));
     }
 
     private void testInviteInstructorAction() {
         ______TS("success: invite an uregistered instructor");
         
-        courseEditPage.clickInviteInstructorLink();
+        courseEditPage.clickInviteInstructorLink(4);
```

Comment:
Just to be super clear, maybe instead of just using `4`, we can use `int unregisteredInstructorId = 4` 

same for the `1` above